pkg/solidity-utils/contracts/test/Weighted5050BasePoolMathMock.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+import { ArrayHelpers } from "../helpers/ArrayHelpers.sol";
+import { WeightedMath } from "../math/WeightedMath.sol";
+import "../math/FixedPoint.sol";
+import "./BasePoolMathMock.sol";
+
+contract Weighted5050BasePoolMathMock is BasePoolMathMock {
```

Comment:
Likewise would like to see block comments on the contracts for why we're testing these particular things. Do we need a Weighted8020BasePoolMathMock? Or a generalized one?

Reply:
I think that the more different invariants we check on rounding tests, the better because `BasePoolMath` depends a little on `computeInvatiant`.